Dockerfile

```diff
@@ -186,6 +186,11 @@ RUN pip3 install "numpy==1.17.3" && \
    pip3 install "pandas==0.25.3" && \
    pip3 install "snakemake==5.8.1"
 
+
+# pip install for modules Ras, NF1, and TP53 Classifiers
```

Comment:
I think you'll want to install these packages into the previous `RUN` command with the multi-line formatting. The multi-line is for convenience/readability but keeping them in the same `RUN` command will put all the python installs on the same layer.

Reply:
Actually it looks like for R we're doing this this way (multiple RUN commands), so never mind on adding these to the above RUN command! I do think the multi-line formatting is easier to read though and would prefer that.

Reply:
ah ok yeah, I can make the edit to make it more readable. Thanks!